core/src/main/java/digitalusage/rogers/com/actors/util/http/TopUpPurchasedClient.java

```diff
@@ -69,8 +69,12 @@ public TopUpPurchasedClient(String url, String table, long ttl, String ucsClient
 
     public static final Function<RetrievePurchasedTopUpsRequest, JsonObject> maptoPurchasedRequest = request -> {
 
+        String genesisApplicationId = request.getContext().getApplicationId();
```

Comment:
There should be no reference to genesis.
```suggestion
        String applicationId = request.getContext().getApplicationId();
```

Reply:
We can change this one -it is the inbound applicationId, but we already have a lot of references to genesis.
I have set 

Client {{genesisApplicationId}} --> GENESIS {{ucsApplicationId}} --> UCS  mapping in conf.
We can rename genesisApplicationId as clientApplicationId

Reply:
would like to know what the config looks like.